staging/src/k8s.io/sample-controller/README.md

```diff
@@ -3,7 +3,7 @@
 This repository implements a simple controller for watching Foo resources as
 defined with a CustomResourceDefinition (CRD).
 
-**Note:** go-get or vendor this package as `k8s.io/sample-controller`.
+:exclamation:**Note:** go-get or vendor this package as `k8s.io/sample-controller`.
```

Comment:
Thanks for your PR, but I don't think this adds extra benefit. The "Note" is already in bold.

/close